src/ursa/cli/__init__.py

```diff
@@ -127,5 +138,185 @@ def version() -> None:
     print(get_version("ursa-ai"))
 
 
+# TODO: add help
+@app.command()
+def serve(
+    host: Annotated[
+        str,
+        typer.Option("--host", help="Bind address.", envvar="URSA_HOST"),
+    ] = "127.0.0.1",
+    port: Annotated[
+        int,
+        typer.Option("--port", "-p", help="Bind port.", envvar="URSA_PORT"),
+    ] = 8000,
+    reload: Annotated[
+        bool,
+        typer.Option(
+            "--reload/--no-reload", help="Auto-reload on code changes."
+        ),
+    ] = False,
+    log_level: Annotated[
+        str,
+        typer.Option(
+            "--log-level",
+            "-l",
+            help="Uvicorn log level: critical|error|warning|info|debug|trace",
+            envvar="URSA_LOG_LEVEL",
+        ),
+    ] = "info",
+    workspace: Annotated[
+        Path, typer.Option(help="Directory to store ursa ouput")
+    ] = Path(".ursa"),
```

Comment:
```suggestion
    ] = Path("ursa_mcp_workspace"),  # or "ursa_workspace" like the cli? # or something else?
```
I'm unclear on what the behavior should be. Will having the same workspace for everyone hitting this endpoint cause issues?